http_logs/operations/default.json

```diff
@@ -544,4 +559,42 @@
           }
         ]
       }
+    },
+    {
+      "name": "range_with_desc_sort",
+      "operation-type": "search",
+      "index": "logs-*",
+      "body": {
+        "size": 100,
+        "query": {
+          "range": {
+            "size": {
+              "gte": 10,
+              "lte": 200
+            }
+          }
+        },
+        "sort": [
+          { "size": "desc" }
+        ]
+      }
+    },
+    {
+      "name": "range_with_asc_sort",
+      "operation-type": "search",
+      "index": "logs-*",
+      "body": {
+        "size": 100,
+        "query": {
+          "range": {
+            "size": {
+              "gte": 10,
+              "lte": 200
```

Comment:
would users ever want to change these values? might be good to include a way to parameterize it in the future and use a workload.py to update it.

Reply:
That would be helpful to other users.